local-cli/runAndroid/runAndroid.js

```diff
@@ -92,17 +92,20 @@ function buildAndRun(args, reject) {
   }
 
   try {
-    const packageName = fs.readFileSync(
-      'app/src/main/AndroidManifest.xml',
+    const manifest = fs.readFileSync(
+      'app/build/intermediates/manifests/full/debug/AndroidManifest.xml',
       'utf8'
-    ).match(/package="(.+?)"/)[1];
+    );
+
+    const applicationId = manifest.match(/package="(.+?)"/)[1];
```

Comment:
this is actually the same match, than in the original version, I just changed the AndroidManifest location.